Source/WebCore/dom/TrustedType.cpp

```diff
@@ -173,6 +173,32 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+ExceptionOr<String> trustedScriptCompliantStringFromVariant(ScriptExecutionContext& scriptExecutionContext, std::variant<RefPtr<TrustedScript>, String>&& input, const String& sink)
```

Comment:
This function doesn’t need `FromVariant` in its name, because of how overloading works in C++. Can we just make it an overload for `trustedScriptCompliantString`?